docs/source/tutorials.md

```diff
@@ -19,11 +19,13 @@ JAX_basic_text_classification
 JAX_examples_image_segmentation
 JAX_Vision_transformer
 JAX_machine_translation
+<<<<<<< HEAD
```

Comment:
Stray git conflict marker; we need to remove this.

Reply:
done.